cmd/shutdown_windows.go

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/dapr/cli/pkg/print"
+	"golang.org/x/sys/windows"
+)
+
+func setupShutdownNotify(sigCh chan os.Signal){
+	//This will catch Ctrl-C
+	signal.Notify(sigCh, syscall.SIGTERM, syscall.SIGINT)
+
+	// Unlike Linux/Mac, you can't just send a SIGTERM from another process
+	// In order for 'dapr stop' to be able to signal gracefully we must use a named event in Windows
```

Comment:
Ha! So, that is why we see process leaks on Windows.

Reply:
See above.